.github/workflows/linux.yml

```diff
@@ -76,20 +76,36 @@ jobs:
       run: |
         mpicc -v
         cmake --version
+    - uses: actions/checkout@v2
+      with:
+        repository: kokkos/kokkos
+        ref: 3.7.00
+        path: kokkos
+    - name: install kokkos
+      working-directory: kokkos
+      run: |
+        mkdir build
+        cd build
+        cmake -D BUILD_SHARED_LIBS=ON \
+              -D CMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/kokkos-install \
+              ..
+        make install
     - name: configure
       run: |
         mkdir build
         cd build
         cmake -D CMAKE_BUILD_TYPE=Debug \
               -D DEAL_II_CXX_FLAGS='-Werror -std=c++17' \
               -D DEAL_II_EARLY_DEPRECATIONS=ON \
+              -D DEAL_II_WITH_KOKKOS="ON" \
               -D DEAL_II_WITH_CGAL="ON" \
               -D DEAL_II_WITH_MPI="ON" \
-              -D DEAL_II_WITH_TRILINOS="ON" \
+              -D DEAL_II_WITH_TRILINOS="OFF" \
```

Comment:
If you turn trilinos OFF, there is no point in installing before

Reply:
I reverted the changes to this build now.